src/odemis/gui/cont/acquisition.py

```diff
@@ -1397,9 +1397,11 @@ def __init__(self, tab_data, tab_panel, projectbar_ctrl, calibration_ctrl):
 
         # Path to the acquisition
         # <External storage>/<date>/<project name>/<roa name>
```

Comment:
To have it more generic, how about <External storage> -> <Path to external storage> as it is not hardcoded anymore.